cmake/tribits/python_utils/gitdist.py

```diff
@@ -438,14 +441,18 @@ def getHelpTopicsStr():
 usefulAliasesHelp =r"""
 USEFUL ALIASES:
 
-A few very useful (bash) shell aliases to use along with the gitdist script
-are:
+A few very useful (bash) shell aliases and setup commands to use with gitdist
+include:
 
   $ alias gitdist-status="gitdist dist-repo-status"
   $ alias gitdist-mod="gitdist --dist-mod-only"
   $ alias gitdist-mod-status="gitdist dist-repo-status --dist-mod-only"
+  $ alias gitdist-repo-versions="gitdist --dist-no-color log -1 \
+    --pretty=format:\"%h [%ad] <%ae>%n%s\" | grep -v \"^$\""
 
-(i.e. add these to your ~/.bash_profile file.)
+These are added by sourcing the provided file 'gitdist-setup.sh' as well as
+some useful commandline tab completion (which be sourced in you
```

Comment:
"which should be sourced in your"?

Reply:
Fixed in TriBITSPub/TriBITS@d96f8351703a595524e5ebd76c14e5e6ba49754a.  Thanks!